### DISCUSSION TITLE
Disable sending on enter?

### DISCUSSION THREAD

**Discussion by skobkin** [open-webui/open-webui#7352] [Q&A]
Is there an option to disable sending reply on "Enter"?

On the desktop you can avoid that by using `Shift` + `Enter`. But if you're on mobile device, it's possible only if you're using specialized keyboard like [Hacker's Keyboard](https://play.google.com/store/apps/details?id=org.pocketworkstation.pckeyboard) which is good in terminal, but not very well suited for everyday use with smart autocomplete and so on.

**Comment by tjbck** [comment]
Fixed on dev!

**Reply by emptiestplace** [reply]
I believe this change is in f9e2496 - if so, it looks like you've improved handling for mobile devices specifically. I'm wondering if it'd be possible to also extract part of this into a user preference? I know I can shift+enter to insert newlines, but switching between different interfaces, I sometimes forget (or I fail at holding shift when I'm doing it repeatedly), and it's a bit frustrating having to abort / fix.

Apologies if there is already a "send on enter" preference, it was the first thing I looked for and I haven't been able to find anything.
